config/storybook/decorators.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Grid, Paper, Table, TableRow, TableCell, TableHead, TableBody } from "@material-ui/core";
+
+
+
+/** A custom "CombinationRender" for react-storybook-addon-props-combinations addon */
```

Comment:
The TypeDoc doesn't match the name of the component.

Reply:
fixed

Reply:
fixed now